test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -67,12 +96,18 @@
                             (metrics/host-metric-name "localhost" "foo")
                             (metrics/gauge 2))
           (let [resp (http-client/get
-                      (java.net.URLDecoder/decode
                        (str "http://localhost:8180/metrics/v1/mbeans/"
-                            "pl.foo.reg:name=puppetlabs.localhost.foo")))
+                            (codec/url-encode "pl.foo.reg:name=puppetlabs.localhost.foo")))
```

Comment:
The original bit of the test seemed backwards to me, it was running a URL **decode** on `pl.foo.reg:name=puppetlabs.localhost.foo` before sending it to the HTTP GET method, which would undo percent-encoding.